package.json

```diff
@@ -40,6 +40,7 @@
   },
   "devDependencies": {
     "chalk": "^2.4.2",
+    "rollup": "^1.32.1",
```

Comment:
Use `rollup` v2.

Reply:
This would require going all the way to Node.js 10. Is that acceptable?

Reply:
Yes, Node.js 8 is no longer supported by the Node.js team.